MMCoreJ_wrap/MMCoreJ.i

```diff
@@ -642,6 +642,16 @@
 %}
 
 %typemap(javacode) CMMCore %{
+   private Boolean includeSystemStateCache_ = true;
+
+   public Boolean getIncludeSystemStateCache() { 
+      return includeSystemStateCache_;
+   }
+   public void setIncludeSystemStateCache(Boolean state) {
+      includeSystemStateCache_ = state;
+   }
```

Comment:
Sorry I didn't catch this earlier, but all 3 occurrences of `Boolean` should be `boolean` here. I don't think there is a benefit to using the boxed type, and in fact there is the danger of somebody passing `null` for a `Boolean` parameter (which would throw NPE).

Reply:
Done.